packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorModule.m

```diff
@@ -99,6 +99,8 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
           [reportableFaces addObject:[encoder encode:face]];
         }
       }
+
+      CGImageRelease(cgImage);
```

Comment:
@sjchmiela I think it's better to do the release here, after we check the faces.